modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/AbstractSAMLSSOTestCase.java

```diff
@@ -393,7 +393,17 @@ public void deleteApplication(String appName) throws Exception {
         return claimMappingList.toArray(new ClaimMapping[claimMappingList.size()]);
     }
 
-    public SAMLSSOServiceProviderDTO createSsoServiceProviderDTO(SAMLConfig config) {
+    public SAMLSSOServiceProviderDTO createSsoSPDTOForIdPInit(SAMLConfig config){
```

Comment:
Add java doc comments for all the public methods.

Reply:
Added the java doc for the newly added public method.